### DISCUSSION TITLE
Can Glances version number be displayed in web server mode?

### DISCUSSION THREAD

**Discussion by AnthonyBe** [nicolargo/glances#2857] [Q&A] [answered]
Is there any way to have the Glances version number displayed in the Glances webserver somewhere? If not by default, having as an optional attribute to add to header (https://glances.readthedocs.io/en/latest/aoa/header.html) would be great?

**Comment by nicolargo** [comment, marked as answer]
Yes, just click on the '**h**' hotkey and badam:

![image](https://github.com/nicolargo/glances/assets/776747/f203c43e-b4b8-48b2-a2a0-044ca55d16e0)

**Reply by AnthonyBe** [reply]
/facepalm

I thought the toggle hotkeys only worked via terminal window. I had no idea that they ALSO worked via webserver interface!!!

Thank you!!

**Reply by AnthonyBe** [reply]
That said, it would be nice to have it on the main screen in the header so I don't have to toggle to see it :)
